completions/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2020  Matthew Glazar
+# See end of file for extended copyright information.
+
+cmake_minimum_required(VERSION 3.10)
+project(quick-lint-js-completions)
```

Comment:
Why make a `project`? Let's inherit the `project(quick-lint-js)` from `/CMakeLists.txt`.